CONTRIBUTING.md

```diff
@@ -187,25 +198,48 @@ Additional commits might be added to `master` branch between tagged-releases, so
 
 ### Setting-up Your Fork for Contributions
 
-If you intend to contribute contents to this project, you'll need to carry out some additional set-up steps in your local fork in order to be able to contribute to the _upstream_ repository:
+If you intend to contribute contents to this project, you'll need to first clone the repository on GitHub, create a local copy of your fork, and then carry out some additional set-up steps in your local fork in order to be able to contribute to the _upstream_ repository.
+You'll need to create a GitHub account, if you don't already have one.
+
+1. **FORK THE REPOSITORY** — Log-into GitHub and browse to the landing page of the official PureBasic repository:
+
+    - https://github.com/fantaisie-software/purebasic
+
+    then click on the <kbd>Fork</kbd> button that appears on the right side at beginning of the page (you need to be logged-in to see this option).
+    For more info on forking, see [GitHub Docs » Fork a repo][Fork a repo].
+
+2. **CLONE YOUR FORK LOCALLY** — You then need to _clone_ your fork locally, i.e. make a copy on your machine.
+Open up the Git Bash and type:
+
+    ```bash
+    $ git clone https://github.com/<your-GitHub-account>/purebasic.git
```

Comment:
I know `<your-GitHub-account>` is the the placeholder syntax I used on #144 however it doesn't match the one used for `your-branch-name` below at step `2. **BRANCH**`

Reply:
I'm not sure what you mean, cloning is an operation applying to a repository, not to a specific branch. Your example was correct. 

Do you mean that also the other example (the one about checking out a branch) should use angle brackets to indicate it's a placeholder? If that's the case, I don't really think we need to, for they are difference cases — the former can only be replaced by the valid username, whereas in the latter the example provided would be a valid branch name (it's an example, not a placeholder).

Reply:
> Do you mean that also the other example (the one about checking out a branch) should use angle brackets to indicate it's a placeholder? If that's the case, I don't really think we need to, for they are difference cases — the former can only be replaced by the valid username, whereas in the latter the example provided would be a valid branch name (it's an example, not a placeholder).

I said both are placeholder because they can both be replaced with many values. (Even if the user has no choice for the first) 
That why I say both must have the same format. Either both have angle brackets or neither must have them.

Reply:
> That why I say both must have the same format. Either both have angle brackets or neither must have them.

But URLs can't contain angle brackets, whereas shell commands can. It's oranges and apples IMO. In the worst case scenario the end user will end up creating a custom branch named `your-branch-name`, which is fine in terms of creating a PR. 

The real question is whether someone who can't work out the sample command is ready to embark in Git (which is notoriously unfriendly to learn), or if he/she should become more confident with concepts like URLs, CLI tools and commands, etc. The KISS rule states that we should strive to make things simple, but not too simple as to go below the threshold of minimum requirements.

But then, if the issue is felt strongly I can just enclose the second placeholder in angle brackets too, after all the same principle of extrapolation by examples should apply. I just don't see the need for a standard convention across different example types.